js/skripts.js

```diff
@@ -28,20 +28,20 @@ function perimetrs(m1,m2,m3){
 }
 //vēl viens variants perimetra aprēķināšanā
 function perimeter() {
-    var m1 = document.getElementById("mala1").value;
-    var m2 = document.getElementById("mala2").value;
-    var m3 = document.getElementById("mala3").value;
-    var perimeter = parseFloat(m1) + parseFloat(m2) + parseFloat(m3);
+    var malas =nolasa();
+    var perimeter = malas.m1 + malas.m2 + malas.m3;
     var result = document.getElementById("result");
     result.innerHTML = "Perimetrs ir " + perimeter;
 }
 //šeit no pusperimetra var aprēķināt laukumu
 function laukums(){
-    var m1 = document.getElementById("mala1").value;
-    var m2 = document.getElementById("mala2").value;
-    var m3 = document.getElementById("mala3").value;
-    var pusper=(m1+m2+m3)/2;
-    var tlaukums=Math.sqrt(pusper*(pusper-m1)*(pusper-m2)*(pusper-m3));
+    var malas =nolasa();
```

Comment:
Vajag pārbaudīt vai nolasa() neatgrieza false, ja false, tad nevajag neko rēķināt